features/guidance/turn.feature

```diff
@@ -940,12 +940,12 @@ Feature: Simple Turns
 
         And the ways
             | nodes | name          | highway   | oneway |
-            | ab    | Molkenmarkt   | secondary | yes    |
+            | ab    | Stralauer Str | secondary | yes    |
```

Comment:
at the intersection itself it is modelled as Stralauer Str in the data... One could argue that the modeling of this intersection which has Molkenmarkt on multiple of the roads is more than strange, though. I'd question whether the name Mokenmarkt in between is acutally referring to the roads, but rather to the space in between.